subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -102,8 +102,8 @@ include::sample[dir="snippets/testing/test-suite-plugin/kotlin",files="build.gra
 ====
 
 <1> Configure all test suites for this project.
-<2> Configure the built-in `test` suite.  This suite is automatically created for backwards compatibility.  By convention, the built-in `test` suite will use JUnit4.
-<3> Declare this test suite uses JUnit Jupiter.  The framework's dependencies are automatically included.  It is not necessary to explicitly name or configure the built-in `test` suite if JUnit4 is desired.
+<2> Configure the built-in `test` suite.  This suite is automatically created for backwards compatibility.  You *must* specify a testing framework to use in order to run these tests (e.g. JUnit 4, JUnit Jupiter).
```

Comment:
Do you really need to specify `JUnit Jupiter` or isn't it `JUnitPlattform` instead?

Reply:
We settled on platform as the correct choice here, but it's confusing enough (as platform is used elsewhere) that we added a section in the docs: https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html#differences_between_similar_methods_on_jvmtestsuite_and_test_task_types

Reply:
@tresat I'm wondering how would you define a custom `JUnit Platform` based framework then, or what would you do if you have mixed Jupiter and Spock tests?

Reply:
The framework selection just adds dependencies.  So if you want to run a combination of JUnit 5 and Spock tests, you would  call `useJUnitJupiter()` to add the aggregator dep which adds the JUnit 5 engine and the test runner and then additionally add the `spock-core` dependency to the suite write Spock tests.  Not the most intuitive, but we're imagining a single testing framework per suite is the typical usage.

We went back and forth over naming the method platform vs. jupiter, and decided the name of the library you write the tests in was most clear.

Reply:
What about using spek would you still have to add jupiter?

Reply:
We didn't set up a framework method for spek yet.  It would depend on what dependencies we include when we do.

Reply:
I just meant what would you have to do to use an in-house test engine based on the JUnit Platform, where we don't have a special method?

Reply:
Just add dependencies to the suite directly, like this:

```
testing {
    suites {
        test {
            dependencies {
                implementation('my:test-engine:1.0')
            }
        }
    }
}
```

Reply:
Ok, so the test suites imply a `useJunitPlatform()` on the individual `Test` tasks.

Reply:
ATM that is all they do, this will likely change as functionality is added to test suites, so we don't want the concept to thought of in only this strict narrow sense, even if it is true ATM.

The last `useXYZ()` call will win, with no error (provided `options` aren't configured in between calls, that's another story).  See `JUnitJupiterOptionsIntegrationTest#options for test framework are respected for JUnitJupiter for custom test suite with changing test framework()` for an example.